website/docs/d/private_link_service.html.markdown

```diff
@@ -0,0 +1,83 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_private_link_service"
+sidebar_current: "docs-azurerm-datasource-private-link-service"
+description: |-
+  Use this data source to access information about an existing Private Link Service.
+---
+
+# Data Source: azurerm_private_link_service
+
+Use this data source to access information about an existing Private Link Service.
+
+
+## Private Link Service Usage
+
+```hcl
+data "azurerm_private_link_service" "example" {
+  name                = "acctestpls"
+  resource_group_name = "acctestRG"
+}
+
+output "private_link_service_id" {
+  value = "${data.azurerm_private_link_service.example.id}"
+}
+```
+
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the private link service.
+
+* `resource_group_name` - (Required) The name of the resource group in which the private link service resides. Changing this forces a new resource to be created.
+
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `location` - Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.
+
+* `alias` - The alias of the private link service.
+
+* `auto_approval_subscription_ids` - A list of subscription(s) globally unique identifiers that will be auto approved to use the private link service.
+
+* `visibility_subscription_ids` - A list of subscription(s) globally unique identifiers(GUID) that will be able to see the private link service.
+
+* `primary_nat_ip_configuration` - A `primary_nat_ip_configuration` block as defined below.
+
+* `secondary_nat_ip_configuration` -  `secondary_nat_ip_configuration` blocks as defined below.
+
+* `load_balancer_frontend_ip_configuration_ids` - A list of `Standard` Load Balancer resource IDs.
```

Comment:
COuld we elaborate on what the loadbalancer is used for?

Reply:
Fixed.